orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -154,8 +205,8 @@ def _initialize_module_gradient_graph_builder(self):
 
     def _get_inference_graph_and_init_gradient_graph_builder(self, *inputs, **kwargs):
         input_names, dynamic_axes, self._input_names_require_grad = \
-                _extract_input_information(self._original_module, *inputs, **kwargs)
-        self._onnx_inference = self._get_forward_graph(input_names, dynamic_axes, *inputs, **kwargs)
+                _parse_inputs_for_onnx_export(self._original_module, *inputs, **kwargs)
```

Comment:
Minor comment. This line is not required. But it is a very minor comment. We can get it in the next PR.

Reply:
Good catch. I have another PR to go. I can remove there and save some CI time. This is dead code and wont hurt going in now

Reply:
Being addressed at https://github.com/microsoft/onnxruntime/pull/6365